plugin/cancel/cancel.py

```diff
@@ -45,7 +45,8 @@ def pre_phrase(phrase: Phrase):
 
     # Check if the phrase is before the threshold
     if ts_threshold != 0:
-        start = getattr(words[0], "start", phrase["_ts"])
+        # NB: mimic() and Dragon don't have this key.
+        start = getattr(words[0], "start", None) or getattr(phrase, "_ts", ts_threshold)
         phrase_starts_before_threshold = start < ts_threshold
```

Comment:
Can they both be None? If so will None < <ts_threshold, ie some int> throw an error?

Reply:
They can. That's why we have `ts_threshold` as the last default. That means that `delta` is zero and nothing happens.

Reply:
Ah! Right. I'm not used to `getattr`